src/containers/genePage/alleleTable.js

```diff
@@ -79,27 +79,28 @@ const AlleleTable = ({ isLoadingGene, gene, geneId}) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [resolvedData, allelesFiltered.data, alleleIdsSelected, setAlleleIdsSelected]);
 
-  const selectRow = useMemo(() => ({
-    mode: 'checkbox',
-    clickToSelect: true,
-    hideSelectColumn: true,
-    selected: alleleIdsSelected,
-    onSelect: (row) => {
-      const alleleIdRow = row.id;
-      setAlleleIdsSelected(alleleIdsSelectedPrev => {
-        if (alleleIdsSelectedPrev.includes(alleleIdRow)) {
-          const indexAlleleId = alleleIdsSelectedPrev.indexOf(alleleIdRow);
-          return [
-            ...alleleIdsSelectedPrev.slice(0, indexAlleleId),
-            ...alleleIdsSelectedPrev.slice(indexAlleleId + 1)
-          ];
-        } else {
-          return [...alleleIdsSelectedPrev, alleleIdRow];
-        }
-      });
-    },
-    style: { backgroundColor: '#ffffd4' },
-  }), [alleleIdsSelected, setAlleleIdsSelected]);
+  //uncomment when sequence viewer is working again
+  //const selectRow = useMemo(() => ({
+  //  mode: 'checkbox',
+  //  clickToSelect: true,
+  //  hideSelectColumn: true,
+  //  selected: alleleIdsSelected,
+  //  onSelect: (row) => {
+  //    const alleleIdRow = row.id;
+  //    setAlleleIdsSelected(alleleIdsSelectedPrev => {
+  //      if (alleleIdsSelectedPrev.includes(alleleIdRow)) {
+  //        const indexAlleleId = alleleIdsSelectedPrev.indexOf(alleleIdRow);
+  //        return [
+  //          ...alleleIdsSelectedPrev.slice(0, indexAlleleId),
+  //          ...alleleIdsSelectedPrev.slice(indexAlleleId + 1)
+  //        ];
+  //      } else {
+  //        return [...alleleIdsSelectedPrev, alleleIdRow];
+  //      }
+  //    });
+  //  },
+  //  style: { backgroundColor: '#ffffd4' },
+  //}), [alleleIdsSelected, setAlleleIdsSelected]);
```

Comment:
@adamgibs you might be able to just leave the method uncommented and just comment out below

Reply:
ahh ok